src/Inputs/Checkboxes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jeremeamia\Slack\BlockKit\Inputs;
+
+use Jeremeamia\Slack\BlockKit\Exception;
+use Jeremeamia\Slack\BlockKit\Partials\Option;
+
+class Checkboxes extends InputElement
+{
+
+    use HasConfirm;
+
+    private const MIN_OPTIONS = 1;
+    private const MAX_OPTIONS = 10;
+
+    /** @var array|Option[] */
+    private $options = [];
+
+    /** @var array|Option[] */
+    private $initialOptions = [];
+
+    public function addOption(Option $option, bool $isInitial = false): Checkboxes {
+
+        $this->options[] = $option;
+
+        if ($isInitial) {
+            $this->initialOptions[] = $option;
+        }
+
+        return $this;
+
+    }
+
```

Comment:
You can use this method for setting `initialOptions` (borrowed from an existing element):
```php
    /**
     * @param array $options
     * @return self
     */
    public function initialOptions(array $options): self
    {
        foreach ($options as $name => $value) {
            $option = new Option((string) $name, (string) $value);
            $option->setParent($this);
            $this->initialOptions[] = $option;
        }

        return $this;
    }
```